specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-04-01-preview/cosmos-db.json

```diff
@@ -8861,6 +9137,322 @@
         }
       }
+    },
+    "ServiceResource": {
+      "description": "Service cluster response resource.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The unique resource identifier of the ARM resource."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The service name."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of Azure resource."
+        },
+        "properties": {
+          "type": "object",
+          "x-ms-client-flatten": true,
+          "description": "The properties of a Data Transfer cluster.",
+          "$ref": "#/definitions/ServiceClusterProperty"
+        }
+      },
+      "required": [
+        "properties"
+      ],
+      "x-ms-azure-resource": true
+    },
+    "ServiceClusterProperty": {
+      "description": "Describes the service cluster response property.",
+      "type": "object",
+      "properties": {
+        "status": {
+          "readOnly": true,
+          "type": "string",
+          "enum": [
+            "Creating",
+            "Running",
+            "Updating",
+            "Deleting",
+            "Error",
+            "Stopped"
+          ],
+          "description": "Describes the status of a Data Transfer job.",
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "ClusterStatus"
+          }
+        },
+        "creationTime": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "Time of the last state change (ISO-8601 format)."
+        },
+        "instanceSize": {
+          "type": "string",
+          "enum": [
+            "Cosmos.D4s",
+            "Cosmos.D8s",
+            "Cosmos.D16s"
+          ],
+          "description": "Instance type for the Data Transfer cluster.",
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "ClusterInstanceSize"
+          }
+        },
+        "instanceCount": {
+          "description": "Instance count for the Data Transfer cluster.",
+          "type": "number"
+        },
+        "serviceType": {
+          "type": "string",
+          "description": "The type of Azure resource."
+        },
+        "locations": {
+          "type": "array",
+          "readOnly": true,
+          "description": "An array that contains all of the locations for the data transfer cluster.",
+          "items": {
+            "$ref": "#/definitions/RegionalServiceLocation"
+          }
+        }
+      },
+      "required": [
+        "instanceSize",
+        "instanceCount"
+      ]
+    },
+    "RegionalServiceLocation": {
+      "description": "Resource for a regional service location.",
+      "type": "object",
+      "properties": {
```

Comment:
This will also contain sqlDedicatedGatewayEndpoint for sqlDedicatedGateway scenarios.